packages/adminrouter/extra/src/README.md

```diff
@@ -660,29 +894,28 @@ with the line length limit hard-coded to 4096 bytes.
 ##### NGINX
 The NGINX subprocess is different from others in regard to its lifetime. Pytest
 fixture that pulls it into the test is module-scoped by default. If there is a
-need to have custom lifetime or just single-test scoped lifetime, then it's
-necessary to use `nginx_class` fixture instead of simple `master_ar_process` or
-`agent_ar_process` ones.
+need to have custom lifetime, then it's necessary to use `nginx_class`
+fixture which provides maximum flexibility.
 
 NGINX instances have the `.make_url_from_path` method which is a convenient way
-to generate AR URLs for tests. It uses exhibitor endpoint as it's present in
-all DC/OS configurations and uses auth features as well.
+to generate AR URLs for tests. User does not have to worry whether this is
+agent or master instance nor know the TCP port that given instance listens on.
 
-#### pytest tests
-Directly from the fact that Admin Router subprocess fixture is module scoped,
-stems some of the current structure of the tests directory. There are three
-different files where tests reside:
+#### AR instance reuse
+Some of the tests can share the same AR instance as it is not being mutated by
+them. There are currently three files which contain:
```

Comment:
The whole paragraph would like like this:
```
Some of the tests can share the same AR instance as it is not being mutated by
them. Three files which contain:
* `test_agent.py`: all the tests related to agent Admin Router, which do not
```
I am not sure I see what you mean her - it does not make sense like this.